.vscode/extensions.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
VSCode extensions should not be part of the project. Please remove this file.

Reply:
https://github.com/nrwl/nx/blob/master/packages/workspace/src/generators/workspace/files/__dot__vscode/extensions.json__tmpl__

Are we sure we want to remove this?  This is the source of it.